tests/test_features/test_jointplot.py

```diff
@@ -0,0 +1,65 @@
+# tests.test_features.test_jointplot
+# Test the JointPlotVisualizer
+#
+# Author:   Prema Damodaran Roman
+# Created:  
```

Comment:
For this line, you can go ahead and put in a  timestamp formatted like as follows: Mon Apr 10 08:19:54 2017 -0400.